### PR TITLE
Adding a basic controller

### PR DESCRIPTION
**Pull request by gbenhaim**
* The controller will watch for events for Job objects.
* Add logger-handler which print information about events that are
  being handled by the controller.
* Add job manifest for testing.
* Add controller Dockerfile
* Add controller manifest
* Update docs.

Signed-off-by: gbenhaim <galbh2@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
...
+
+mkvirtualenv -p /opt/rh/rh-python36/root/usr/bin/python kubespresso
+pip install -U pip
+pip install -r requirements.txt
+```
+
+## Running the controller locally
+
+* Setup the virtualenv
+* Make sure that you are login to the cluster with a user that has cluster-admin role
```
Comment (danielBelenky) [inline README.md]: s/login/logged in

Comment (gbenhaim) [reply README.md]: fixed

File: manifests/controller.yaml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    metadata:
+      name: kubespresso-controller
+      labels:
+        kubespresso.io: kubespresso-controller
+    spec:
+      serviceAccountName: kubespresso-controller
+      containers:
+        - name: kubespresso
+          command: ['python', '/kubespresso.py']
+          image: quay.io/gbenhaim/kubespresso:1.0.0
```
Comment (danielBelenky) [inline manifests/controller.yaml:51]: Can you open an org for this image? It will look better when presenting

**Review by danielBelenky — APPROVED** [review]
Merging to set automated builds in quay.io

**Pull request merged**